eslint.config.mjs

```diff
@@ -27,6 +27,10 @@ const config = [
 		languageOptions: {
 			ecmaVersion: 'latest',
 			sourceType: 'module',
+			globals: {
+				...globals.browser,
+				...globals.node,
+			},
```

Comment:
All files (not only from `core`) should not complain about `document`, `requestAnimationFrame()`, etc. I kept the other settings for now. We can revise them later.